unittests/AST/ASTImporterTest.cpp

```diff
@@ -271,6 +271,11 @@ class TestImportBase : public ParameterizedTestsFixture {
   }
 };
 
+template <typename T> RecordDecl *getRecordDecl(T *D) {
+  auto *ET = cast<ElaboratedType>(D->getType().getTypePtr());
+  return cast<RecordType>(ET->getNamedType().getTypePtr())->getDecl();
+};
```

Comment:
There is a compiler warning because extra `;` here.

Reply:
Thanks, I'll fix it.